src/shogun/evaluation/CrossValidationSplitting.cpp

```diff
@@ -21,6 +21,7 @@ CCrossValidationSplitting::CCrossValidationSplitting(
 	CSplittingStrategy(labels, num_subsets)
 {
 	m_rng = sg_rand;
+	build_subsets();
```

Comment:
a problem here is that we can only call the default ctor from swig, then we may still need to call build_subset from the outside? @karlnapf